src/modules/temperature_compensation/temperature_calibration/task.cpp

```diff
@@ -100,7 +100,7 @@ class TemperatureCalibration
 void TemperatureCalibration::task_main()
 {
 	// subscribe to all gyro instances
-	int gyro_sub[SENSOR_COUNT_MAX] {-1, -1, -1};
+	orb_sub_t gyro_sub[SENSOR_COUNT_MAX] {ORB_SUB_INVALID, ORB_SUB_INVALID, ORB_SUB_INVALID};
 	px4_pollfd_struct_t fds[SENSOR_COUNT_MAX] {};
```

Comment:
Seem to be some remnants to px4_poll and the data struct in many places, should these be removed too ?

Reply:
nope, see above. Subscriber is polling uORB changes